module/index.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
just thinking of typical npm module layout. I can't imagine we'll have any sibling files to this file. @paulirish may have an opinion since he did the last Great Rearrangement, but we can land in the meantime